quartz/styles/base.scss

```diff
@@ -192,6 +192,11 @@ a {
       width: initial;
       margin-top: 2rem;
     }
+    .content-meta {
+      > span {
+        margin-right: 8px;
+      }
+    }
```

Comment:
```suggestion
```

Reply:
You should move this to L59 of the ContentMeta.tsx file instead of leaving it here.